### ISSUE TITLE
camera screen off

### ISSUE THREAD

**Issue by toto6976** [Genymobile/scrcpy#5404]
Hi
s it possible to operate the camera while keeping the screen off ? 

scrcpy --turn-screen-off -b1M --video-source=camera --camera-id=1

**Comment by rom1v** [comment]
Capturing the camera does not turn the screen on.

**Comment by toto6976** [comment]
However when I run the command the screen lights up.
I have lineageos 21 (android 14)

Le ven. 25 oct. 2024, 16:02, Romain Vimont ***@***.***> a
écrit :

**Comment by rom1v** [comment]
I cannot reproduce:

```
scrcpy --video-source=camera
```

does not turn the screen on on any on my devices.

**Comment by rom1v** [comment]
Nope, it does not turn the screen on on my devices with Android 14.

On your device, does it "mirror" the camera correctly?

**Comment by toto6976** [comment]
Yes, the mirrorin it is ok

**Comment by rom1v** [comment]
OK, so its a "bug" (or a "feature") on your device system, which decides to turn the screen on when the camera is captured.

**Comment by toto6976** [comment]
Ah...and what can i do ??
